Zenika/social-video-generatorz
# feat(webapp): integrate videos and event form

## 🤔 Why?

To edit the videos, they must be on the webapp and a form is required to update the data.

## 💻 How?

I've created integrated the videos in the homepage with a link to a dedicated page for each. On the dedicated page, i've build a form that send its values to the props of the player and like that it changes the content of the video.

## ✅ How to validate this PR?

- run `pnpm dev` and go to localhost:3000. You should see the home page with the two preview videos
- Click on `create video`, you should now see the video and its form under it.
- You can edit the default values with yours and it should uptade them on the video instantly
